crates/sui-graphql-rpc/schema/draft_schema.graphql

```diff
@@ -208,6 +208,12 @@ input ObjectFilter {
   owner: SuiAddress
   objectIds: [SuiAddress!]
   objectKeys: [ObjectKey!]
+
+  # Enhancement (post-MVP), compound filters.  Compound filters are
+  # exclusive (must be the only filter set if they are used).
+  any: [ObjectFilter]
+  all: [ObjectFilter]
+  not: ObjectFilter
```

Comment:
@longbowlu, @bmwill, @gegaowp -- I realised we talked about transactions and events, but not objects when we discussed re-introducing  compound filters.  Consider this PR an RFC on including compound filters for Objects, which I think has similar utility to the events case.  Does your reduced concern about how compound queries would affect query complexity extend to objects as well?

Reply:
I think the complexity of compound queries on objects is similar to events and lower than that of transactions, so putting this also as post-MVP enhancement sgtm.